x-pack/plugins/cases/server/features.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+import { KibanaFeatureConfig } from '../../features/common';
+import { DEFAULT_APP_CATEGORIES } from '../../../../src/core/server';
+
+import { APP_ID } from '../common/constants';
+
+/**
+ * The order of appearance in the feature privilege page
+ * under the management section. Cases should be under
+ * the Actions and Connectors feature
+ */
+
+const FEATURE_ORDER = 3100;
+
+export const getCasesKibanaFeature = (): KibanaFeatureConfig => ({
+  id: APP_ID,
+  name: i18n.translate('xpack.cases.features.casesFeatureName', {
+    defaultMessage: 'Cases',
+  }),
+  category: DEFAULT_APP_CATEGORIES.management,
+  app: [],
+  order: FEATURE_ORDER,
+  management: {
+    insightsAndAlerting: [APP_ID],
+  },
+  cases: [APP_ID],
+  privileges: {
+    all: {
+      cases: {
+        all: [APP_ID],
+      },
+      management: {
+        insightsAndAlerting: [APP_ID],
+      },
+      savedObject: {
+        all: [],
+        read: [],
+      },
+      ui: ['crud_cases', 'read_cases'],
+    },
+    read: {
+      cases: {
+        all: [APP_ID],
+      },
```

Comment:
Should this be changed?

```suggestion
      cases: {
        read: [APP_ID],
      },
```

Reply:
Thanks! My mistake 🙂